tests/bug-id.rs

```diff
@@ -11,11 +11,16 @@ use fuel_vm::prelude::*;
 fn check_bug_id_unique() {
     let mut matches = HashSet::new();
     let re = Regex::new(r"BugId::ID\d{3}").expect("failed to create regex");
+    let re_use = Regex::new(r"BugId::\*").expect("failed to create regex");
```

Comment:
Why is this a regex? A string with `.contains` should do the job here.

Reply:
True. Guess I got carried away from using regex haha

https://github.com/FuelLabs/fuel-vm/pull/189/commits/9f6df1d4abc50a3242132e7af506993cad677591